rfc-capi-for-kvm-capk/README.md

```diff
@@ -0,0 +1,449 @@
+# RFC: CAPI for KVM (CAPK)
+
+### Use cases
```

Comment:
Additional use cases:

 * As a GS customer I want to use the Management API to manage clusters and applications via Gitops
 * As a GS customer I want to use the Management API to have more fine grained control over permissions and roles of api users